docs/services/invocation.mdx

```diff
@@ -35,18 +92,38 @@ You should see the response:
 {"greeting":"Hello Restate"}
 ```
 
+</TabItem>
+<TabItem value="java" label="Java">
+
+For example, to invoke the service `org.example.Greeter` method `Greet` using `curl`:
+
+```shell
+curl <RESTATE_INGRESS_URL>/org.example.Greeter/Greet -H 'content-type: application/json' -d '{"name": "Restate"}'
+```
+
+You should see the response:
+
+```json
+{"greeting":"Hello Restate"}
+        ```
+
+    </TabItem>
+</Tabs>
+
 The rules to invoke a service are the following:
 
-* The request path is in the format of `{serviceName}/{methodName}`
+* The request path is in the format of `<SERVICE_NAME>/<METHOD_NAME>`
 * The request method is `POST`
 * The request body is encoded either with:
-    * Json, with the header `Content-Type: application/json`, or with
-    * Protobuf, with the header `Content-Type: application/proto`
+* JSON, with the header `Content-Type: application/json`, or with
+* Protobuf, with the header `Content-Type: application/proto`
```

Comment:
I think these to items are no longer indented correctly (second level of enumeration).